test/__tests__/warehouse.test.js

```diff
@@ -1016,3 +1020,42 @@ describe("Integration options", () => {
     });
   });
 });
+
+describe("validTimestamp", () => {
+  const testCases = [
+    {
+      input: undefined,
+      expected: false,
+    },
+    {
+      input: '-0001-11-30T00:00:00+0000',
+      expected: false,
+    },
+    {
+      input: '2023-06-14T05:23:59.244Z',
+      expected: true,
+    },
+    {
+      input: 'abc',
+      expected: false,
+    },
+    {
+      input: '',
+      expected: false,
+    },
+    {
+      input: '2023-06-14',
+      expected: true,
+    },
+    {
+      input: '05:23:59.244Z',
+      expected: false,
```

Comment:
Can we add these inputs too in testcases?

`../../WEB-INF/web.xml`

`%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216%u002e%u002e%u2216Windows%u2216win%u002ein`

Reply:
Also any more weird ones we found during testing